_posts/en/newsletters/2021-03-10-newsletter.md

```diff
@@ -0,0 +1,202 @@
+      Bitcoin Core.  Using heights would provide a bit more
+      predictability, especially for miners, and would be compatible
+      with other attempts using BIP8.
+
+    - FIXME:harding waiting to understand rusty concern https://hash.social/@harding/105850276827608357
+
+    - *Speed:* the proposal, based on initial discussion from the
+      ##taproot-activation IRC channel, proposes giving miners about
+      three months to lock in taproot and waiting a fixed six months
+      from the start of signal measuring before activation (if lock in
+      is achieved).  Some people have sought either slightly shorter or
+      slightly longer timelines.
+
+    We'll continue tracking the discussion around the various proposals
+    and will summarize any significant progress in future newsletters.
+
+- **Documenting the intention to use and build upon taproot:** In
+  discussion about activation methods, Chris Belcher [noted][flag day
+  belcher] that a large list of software was compiled whose developers
+  stated their intention to implement segwit during the debate around
+  activating that soft fork.  He suggested that a similar list could be
+  created to document for posterity the amount of support taproot has.
+  That way it could be clear that taproot was desired by a large segment
+  of the economy no matter how it ends up being activated.
+
+    Jeremy Rubin [posted][rubin building] to the Bitcoin-Dev mailing
+    list a link to a somewhat related [wiki page][taproot uses] where
+    developers can post links to projects they're building on top of
+    taproot's new proposed features.  This can provide assurance that
+    taproot provides solutions people actually want and is designed in
+    such a way that its features will get used.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+FIXME:jonatack or jnewbery
+
+{% include functions/details-list.md
+  q0="FIXME"
+
+  a0="FIXME"
+
+  a0link="http://example.com/FIXME"
+%}
+
+## FIXME:releases
+
+- FIXME:harding Eclair 0.5.1
+
+- FIXME:harding HWI 2.0.0 RCs
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #20685][] Adds I2P support by using the
+  [I2P SAM protocol][I2P SAM protocol].  This feature has
+  [long been requested][Bitcoin Core #2091] and was only recently made
+  possible by the addition of [addr v2][addr v2 newsletter mention].
```

Comment:
Would it be better to link to the addr v2 topic here, rather than just the PR merge coverage?